README.md

```diff
@@ -20,29 +20,62 @@ See our [Design Philosophy](docs/README.md#design-philosophy).
 Using `react-map-gl` requires `react >= 16.3`.
 
 ```sh
-npm install --save react-map-gl mapbox-gl
+# Using Maplibre
+npm install react-map-gl maplibre-gl
+```
+_or_
+
+```sh
+# Using Mapbox
+npm install react-map-gl mapbox-gl @types/mapbox-gl
```

Comment:
We still use the community types to support all of the older major versions of mapbox https://github.com/visgl/react-map-gl/blob/92706b745257745748d77e2e0a22a1578c05331c/package.json#L54 

I wish there were a better migration path for this kind of thing.. we could be exporting types from `@types/mapbox-gl`. For now, since we still rely on it, I think its appropriate to document and revisit if we move to their native types.